sdk/spring/azure-spring-boot-samples/azure-spring-cloud-appconfiguration-sample/README.md

```diff
@@ -1,6 +1,6 @@
 # Spring Cloud Azure Config Sample client library for Java
 
-This sample describes how to use [spring-cloud-starter-azure-appconfiguration-config](../../azure-spring-cloud-starter-appconfiguration-config/) to load configuration properties from Azure Configuration Service to Spring Environment.
+This sample describes how to use [spring-cloud-starter-azure-appconfiguration-config](../../../appconfiguration/azure-spring-cloud-starter-appconfiguration-config/) to load configuration properties from Azure Configuration Service to Spring Environment.
```

Comment:
Seems like we shouldn't use relative links here.

https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#link-guidelines

https://github.com/Azure/azure-sdk-for-java/pull/13989/files#diff-bbc5dba8ff02aedfeed94fd0f102c996